stdlib/REPL/src/REPL.jl

```diff
@@ -427,16 +428,30 @@ LineEditREPL(t::TextTerminal, hascolor::Bool, envcolors::Bool=false) =
         false, false, false, envcolors
     )
 
-mutable struct REPLCompletionProvider <: CompletionProvider end
+mutable struct REPLCompletionProvider <: CompletionProvider
+    modifier::Symbol
```

Comment:
```suggestion
    shift::Bool
```
It would be valid to combine modifiers, so I think we should add them as-needed individually